src/OrbitSshQt/SftpCopyToLocalOperation.cpp

```diff
@@ -45,6 +47,17 @@ outcome::result<void> SftpCopyToLocalOperation::shutdown() {
     case State::kStarted:
       ORBIT_UNREACHABLE();
     case State::kShutdown:
+    case State::kCloseAndDeletePartialFile: {
+      local_file_.close();
+      bool remove_result = local_file_.remove();
+      if (!remove_result) {
+        ORBIT_ERROR("Unable to remove partially downloaded file %s",
```

Comment:
What happens if we later try to download the file again? Will it try to override the partial file?

Reply:
Yes. In more detail: The situation is a partial file in the cache. Orbit will look in the cache, find the file and try to parse it as an elf file to check whether the build ID of the cache file is matching the modules. The parsing fails with an error message (I dont have the exact wording at hand, but something like: The section list points to sections that are past the end of the file). Therefore the cache search fails and Orbit searches for a file on the instance and then downloads it.

Reply:
Thanks. Makes sense.